backend/src/test/validation/PromotionValidation.test.ts

```diff
@@ -383,13 +386,16 @@ describe('Unit tests for PromotionValidation', function () {
         name: 3,
         placeId: 4,
         userId: false,
+        lat: '34.0',
+        lon: '-43.2',
```

Comment:
Should these be numbers? Or should lines 44-45 be strings? Or does it not matter 😅

Wondering if this is related to the comment you made in `backend/src/main/validation/PromotionValidation.ts`

Reply:
It is! It'll actually be converted to numbers ☺️

Reply:
Oh how come we can't just enforce that the lat and lon will be numbers and cannot be strings?